src/GuiBase/BaseApplication.cpp

```diff
@@ -502,16 +529,16 @@ BaseApplication::~BaseApplication() {
     QDir().rmdir( m_exportFoldername.c_str() );
 }
 
-bool BaseApplication::loadPlugins( const std::string& pluginsPath,
+bool BaseApplication::loadPlugins( const std::string& pluginsPath2,
```

Comment:
It is just a matter of style, but I don't like this name, may use `pluginPathCandidate` or equivalent ?

Reply:
sure this need polishing